setup.py

```diff
@@ -431,7 +453,16 @@ def do_setup():
         long_description=long_description,
         long_description_content_type='text/markdown',
         license='Apache License 2.0',
-        version=version,
+
+        # Astronomer note: we use a new "version epoch" here so that when
+        # people install our release (which is not published on PyPi) they can
+        # keep our version installed, without it pulling in the latest stock
+        # version from PyPi.
+        #
+        # We start at epoch 2 as there is a bug in Pip (7795) where implicit
```

Comment:
Maybe add a direct link to the issue?

Reply:
For some reason my head went "don't put the URL, that will cause a "mention" in the github issue.

Not sure _how_ I thought that might be the case.